force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.js

```diff
@@ -0,0 +1,143 @@
+import { LightningElement, wire, track, api } from "lwc";
+import { NavigationMixin } from "lightning/navigation";
+import { getObjectInfo } from "lightning/uiObjectInfoApi";
+import { ProgressSteps } from "./progressSteps";
+import { NavigationItems } from "./navigationItems";
+import { format } from "c/util";
+
+import SCHEDULE_OBJECT from "@salesforce/schema/ServiceSchedule__c";
+
+import NEW_RECORD_LABEL from "@salesforce/label/c.New_Record";
+import SAVE_LABEL from "@salesforce/label/c.Save";
+import SAVE_NEW_LABEL from "@salesforce/label/c.Save_New";
+
+export default class ServiceScheduleCreator extends NavigationMixin(LightningElement) {
+    @track
+    labels = {
+        newSchedule: NEW_RECORD_LABEL,
+        save: SAVE_LABEL,
+        saveNew: SAVE_NEW_LABEL,
+    };
+
+    @api serviceId;
+
+    _steps = new ProgressSteps()
+        .addStep("", this.labels.newSchedule, new NavigationItems().addNext())
+        .addStep(
+            "",
+            this.labels.reviewSessions,
+            new NavigationItems().addNext().addPrevious()
+        )
+        .addStep(
+            "",
+            this.labels.addParticipants,
+            new NavigationItems().addNext().addPrevious()
+        )
+        .addStep(
+            "",
+            this.labels.reviewSchedule,
+            new NavigationItems()
+                .addNext(this.labels.saveNew, "neutral")
+                .addPrevious()
+                .addFinish(this.labels.save)
+        );
+    _currentStep;
+
+    @wire(getObjectInfo, { objectApiName: SCHEDULE_OBJECT })
+    wireSchedule(result) {
+        if (!result) {
+            return;
+        }
+        if (result.data) {
+            this.formatLabels(result.data);
```

Comment:
I know this pattern makes for clean code but since it is just one line could we not just add what we have in line 59 to line 52.